tests/fixtures/snapshot_date.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+from typing import Any
+
+import pytest
+from syrupy.matchers import path_type
+
+
+def validate_time(timestamp: datetime.datetime | Any):
```

Comment:
Why are we accepting `Any` here if a `ValueError` is thrown if the value is not a `datetime`?

The raised exception should actually be `TypeError` and a more accurate message is 'Timestamp must be datetime'. Format would be more appropriate for a ISO time `str`.